common/src/cellular_common.c

```diff
@@ -1066,11 +1065,6 @@ CellularError_t _Cellular_LibInit( CellularHandle_t * pCellularHandle,
 
     if( cellularStatus != CELLULAR_SUCCESS )
     {
-        if( bPktResponseMutexCreateSuccess == true )
```

Comment:
This PR is most discussed the unit test, I think your suggestion will be much regarding to the bug fix(?). 
Because any changes will affect the unit test result, the following is my suggesting changes:
```
/* Configure the library. */
    if( cellularStatus == CELLULAR_SUCCESS )
    {
        cellularStatus = libOpen( pContext );
    }

    if( cellularStatus != CELLULAR_SUCCESS )
    {
        if( bPktRequestMutexCreateSuccess == true )
        {
            _Cellular_DestroyPktRequestMutex( pContext );
        }

        if( bAtDataMutexCreateSuccess == true )
        {
            _Cellular_DestroyAtDataMutex( pContext );
        }

        if( bLibStatusMutexCreateSuccess == true )
        {
            _Cellular_DestroyLibStatusMutex( pContext );
        }

        if( pContext != NULL )
        {
            /* Clear this mutex because we created it before. */
            _Cellular_CreatePktResponseMutex( pContext );
            _Cellular_FreeContext( pContext );
        }
    }
    else
    {
        *pCellularHandle = pContext;
    }
```
How do you think?

Reply:
I will add it back, there may be something wrong while testing at that time, but it's ok now.
Thanks for the correction.